code/__HELPERS/reagents.dm

```diff
@@ -73,10 +73,10 @@
 /proc/add_chemical_reaction(datum/chemical_reaction/R)
 	if(!GLOB.chemical_reactions_list_reactant_index || !R.required_reagents || !R.required_reagents.len)
 		return
-	var/primary_reagent = R.required_reagents[1]
-	if(!GLOB.chemical_reactions_list_reactant_index[primary_reagent])
-		GLOB.chemical_reactions_list_reactant_index[primary_reagent] = list()
-	GLOB.chemical_reactions_list_reactant_index[primary_reagent] += R
+	var/rand_reagent = pick(R.required_reagents)
```

Comment:
while you're here, can you fix this single letter var `R` please

Reply:
sure